workshop/content/exercises/01-operator-prerequisites.md

```diff
@@ -6,7 +6,7 @@ NextPage: 02-create-cluster
 
 The MySQL operator is installed into a project to monitor requests to create a MySQL cluster. The operator can only be deployed and setup by a cluster admin of the OpenShift cluster.
 
-In this workshop environment, the MySQL operator has been pre-installed into your project ready for use. The user you are running the workshop as, has also been delegated the appropriate roles to allow it to create a MySQL cluster.
+In this workshop environment, the MySQL operator has been pre-installed into your project ready for use. The user you are running the workshop as has also been delegated the appropriate roles to allow it to create a MySQL cluster.
```

Comment:
This comma is unnecessary. Read the sentence aloud; one doesn't pause between "as" and "has". The sentences could be rendered more clearly, if you want to make some change:
"The MySQL cluster operator has been installed and is ready to use. Your user has been granted the appropriate permissions to create a managed MySQL database cluster."

Reply:
Thanks! I just pushed something up. Let me know if this reads a little bit clearer.

Reply:
LGTM. Thank you. :)